### ISSUE TITLE
having issues with items not deleting

### ISSUE THREAD

**Issue by dozer3677** [BinaryWorlds/FacebookClearActivityLog2025#6]
The script isnt deleting certain activities. Example: I commented on a post 3 years ago, and that post has since been deleted. But it still says "You commented on X post" on X date. If I go to manually delete that, it just says "SOMETHING WENT WRONG". When using the script, the script will just get stuck trying to remove that comment/activity 1000 times, instead of ignoring it and moving on to the next one. Entering values in the IGNORED ITEMS script doesn't seem to do anything, as the script will still attempt to remove that stuck activity 1000 times. Any solutions/assistance? Thanks.

**Comment by pilotalejo** [comment]
im stuck with the same it does not ignore the something when wronmg and continue to thenext step
\

**Comment by BinaryWorlds** [comment]
Yes, I’ve seen that too. Once Facebook throws “Something went wrong,” you won’t be able to delete the item manually either. The simplest solution is to refresh the page (F5) and restart the script.
